src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age ? (
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      ) : null}
+      {person.isMarried === false ? (
+        <p className="Person__partner">I am not married</p>
+      ) : (
+        <p className="Person__partner">
+          {person.sex === 'm'
+            ? `${person.partnerName} is my wife`
+            : `${person.partnerName} is my husband`}
+        </p>
+      )}
```

Comment:
1. Better alter tag content, not the tag itself

```suggestion
      <p className="Person__partner">
      {person.isMarried ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}` : 'I am not married'}
      </p>
```

2. Minor - prefer using positive condition for clean code (`person.isMaried ? ...` over `person.isMarried === false ?`)